docs/services/database.md

```diff
@@ -1,7 +1,7 @@
 The Database service allows you to create structured collections of documents, query and filter lists of documents, and manage an advanced set of read and write access permissions.
```

Comment:
let's rename this file to `databases.md` also the folder to `databases` as we have updated database service to `databases` service